src/extendables/User/Minion.ts

```diff
@@ -761,4 +762,29 @@ export default class extends Extendable {
 			addItemToBank(currentCreatureScores, creatureID, amountToAdd)
 		);
 	}
+
+	public resolveAvailableItemBoosts(this: User, monster: KillableMonster) {
+		const boosts = new Bank();
+		if (monster.itemInBankBoosts) {
+			for (const boostSet of monster.itemInBankBoosts) {
+				let highestBoostAmount = null;
+				let highestBoostItem = null;
+
+				// find the highest boost that the player has
+				for (const [itemID, boostAmount] of Object.entries(boostSet)) {
+					const parsedId = parseInt(itemID);
+					if (!this.hasItemEquippedOrInBank(parsedId)) continue;
+					if (!highestBoostAmount || boostAmount > highestBoostAmount) {
```

Comment:
`highestBoostAmoun` and `highestBoostItem` can just be initialized with `0` which would allow you not to have to check if it exists here before comparing

Reply:
I initially did this, but figured negative boosts might a thing but that sounds silly thinking about it again.